EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: <to be assigned>
+title: Giving Ethereum fees to Future Salaries
+author: Victor Porton <porton@narod.ru>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-02-13
+---
+
+## Simple Summary
+Transfer a part of Ethereum transfer/mining fees to Future Salaries contract
+
+## Abstract
+Part of mining/transfer fees to the contract `SalaryWithDAO` to incentivize traders to pay to common good producers. That directly benefits not only common good producers but miners/validators, too.
```

Comment:
The abstract should be a human readable and terse version of the specification..  The content currently here belongs more in the ## Motivation section rather than the abstract.  Since you don't yet have an actual specification, consider just leaving this as
```suggestion
TBD
```